lab/lablib/SoundManager.js

```diff
@@ -110,5 +148,27 @@ export class SoundManager {
     const transport = this.tone.getTransport();
     transport.position = 0;
     transport.start("+0.1", "0:0");
+    this.transport_playing = true;
+  }
+
+  play_sfx(sfx_id) {
```

Comment:
This code is a more flexible version of `play_score`. The only differences are:

- `play_score` forces the transport back to the start. This isn't really necessary, scheduling now + a bit doesn't require changing the transport
- `play_score` clears the timeline of old events. This should be decoupled

Reply:
I don't want refactoring this to hold up the PR, so I added this to my notes